src/components/form/form.styles.ts

```diff
@@ -6,19 +6,161 @@
  * Side Public License, v 1.
  */
 
-import { UseEuiTheme } from '../../services';
+import {
+  UseEuiTheme,
+  shade,
+  tint,
+  darken,
+  transparentize,
+  makeHighContrastColor,
+} from '../../services';
+import { mathWithUnits } from '../../global_styling';
 
 export const euiFormVariables = (euiThemeContext: UseEuiTheme) => {
-  const { euiTheme } = euiThemeContext;
+  const { euiTheme, colorMode } = euiThemeContext;
+  const isColorDark = colorMode === 'DARK';
+  const backgroundColor = isColorDark
+    ? shade(euiTheme.colors.lightestShade, 0.4)
+    : tint(euiTheme.colors.lightestShade, 0.6);
+
+  const controlHeight = euiTheme.size.xxl;
+  const controlCompressedHeight = euiTheme.size.xl;
 
   return {
+    // Sizing
     maxWidth: `${euiTheme.base * 25}px`,
```

Comment:
minor nit: if a consumer overrides all our sizes to use a different unit (e.g. rem instead of px or w/e), preferring `mathWithUnits` supports that more organically

```suggestion
    maxWidth: mathWithUnits(euiTheme.size.base, (x) => x * 25),
```